di-ipv-queue-stub/enqueue-event/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import type { SignedJwtParams } from 'di-stub-oauth-client';
+export type RequestPayload = {
```

Comment:
I find breaking these out into separate types and then combining them at the end makes the code a lot easier to read rather then combing them as one, as it also explains at what times you would get each of these.